README.md

```diff
@@ -110,6 +110,10 @@ nox
 This will launch the testing scenario, described in `noxfile.py`. This scenario includes rebuilding docker images
 for every supported Python version and running all the tests for corresponding Python.
 
+Also, setting up "REUSE_DOCKER_CONTAINERS" environment variable (or running tests in PyCharm) will let tests
+run in docker cache and reuse test containers. But do note that this is recommended for development purposes only,
```

Comment:
Tests are not run in docker cache.
The correct way to say this would be: "Tests reuse already created and initialized containers, which speeds up the testing process".

Reply:
It was "tests run in docker" "cache and reuse container". But ok, I'll rephrase that.